.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: objective-c
-osx_image: xcode9
+osx_image: xcode9.3beta
```

Comment:
travis still in 9.3beta?

Reply:
Probably should drop the "beta", but it will work for now.